### DISCUSSION TITLE
(FIXED) Prisma Python client inserting data into the wrong fields

### DISCUSSION THREAD

**Discussion by 8auu** [RobertCraigie/prisma-client-py#460] [General]
Edit: Fixed the problem, I'm pretty sure the commands to update schema were not working, so I had to copy it in from my web app and generate the prisma client that way.

Hello,

I'm having some issues with the Python Prisma client (version: 0.6.6). I have some data which I'm trying to insert into my database but it seems to be ignoring the keys I specify and puts the data into random columns...

example: `logChannelId` I set to `"3"` but in the database it is inserted as `"4"`?

I am getting no errors in my console. I have ran the following commands to update my schema (all seems okay):
```
prisma generate
prisma db pull
prisma migrate dev
```

schema.prisma
```
model serververification {
  id             Int     @id @default(autoincrement())
  guildId        String  @db.VarChar(20)
  enabled        Boolean @db.Bit(1)
  logChannelId   String? @db.VarChar(20)
  panelMessageId String? @db.VarChar(20)
  panelChannelId String? @db.VarChar(20)
  verifiedRoleId String? @db.VarChar(20)

  @@index([guildId], map: "guildId_fkey")
}
```
```
        await self.client.prisma.serververification.create(data={
            "guildId": "1",
            "enabled": True,
            "logChannelId": "3",
            "panelMessageId": "4",
            "panelChannelId": "5",
            "verifiedRoleId": "6",
        })
```
Inserted data in database...
![image](https://user-images.githubusercontent.com/54277586/182839581-5b910ed4-b131-411d-a4c6-b70d2489d6f5.png)

**Comment by riverallzero** [comment]
Hi, @8auu 
how about try like this?
<pre><code>await self.client.prisma.serververification.create({
  data: {
    guildId: "1",
    enabled: True,
    logChannelId: "3",
    panelMessageId: "4",
    panelChannelId: "5",
    verifiedRoleId: "6",
  },
});</code></pre>
